packages/node/test/test-constants.jest.ts

```diff
@@ -1,8 +1,8 @@
-export const A_MNEMONIC =
-  "maximum guide cement kiwi lobster skate immense expand spot crew zoo ring";
+export const A_EXTENDED_KEY =
```

Comment:
I still think this shoud be `A_EXTENDED_PRIVATE_KEY` or `A_XPRIV`, etc

Reply:
oh good catch, it should be